subprojects/file-watching/src/integTest/groovy/org/gradle/internal/watch/NoDaemonFilesystemWatchingIntegrationTest.groovy

```diff
@@ -16,13 +16,14 @@
 
 package org.gradle.internal.watch
 
-
```

Comment:
💄 Extra newline

Reply:
Interesting, this was not automatically the format IDEA followed?

Reply:
Now I'm conflicted... there is a single newline here, that should be the standard

Reply:
Ah, sorry I looked wrong. Removing that newline is good!